model/registry/system.yaml

```diff
@@ -254,3 +254,22 @@ groups:
         brief: >
           The process state, e.g., [Linux Process State Codes](https://man7.org/linux/man-pages/man1/ps.1.html#PROCESS_STATE_CODES)
         examples: ["running"]
+  # system.linux.* attribute group
+  # system.linux.memory.slab* attribute group
+  - id: registry.system.linux.memory.slab
```

Comment:
nit: I feel like it's too narrow group. Maybe it makes sense to have `system.linux.memory` grouping? So that if someone will add more attributes he could reuse this group.

Reply:
I agree, and this is even disallowed according to the rules here: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#metric-attributes

The attributes should be under `system.linux.memory`.

Reply:
Sounds good to me. Indeed, there is another Linux memory metric (Kernel) that could benefit from the `reclaimable` state attribute too.

Reply:
Attributes moved to `system.linux.memory` in https://github.com/open-telemetry/semantic-conventions/pull/1078/commits/6b01a6fbcd79fed011cb7eae3be9f474dbe7e2c9